front-end/src/app/shared/resolvers/transaction.resolver.spec.ts

```diff
@@ -106,7 +106,7 @@ describe('TransactionResolver', () => {
     });
   });
 
-  it('should attach child for transaction with dependent child transaction type', () => {
+  xit('should attach child for transaction with dependent child transaction type', () => {
```

Comment:
This unit test is broken.  The expect() call never registers with Karma, and attempts to make it asynchronous did not work